### ISSUE TITLE
Multiple Cluster Selection Bug

### ISSUE THREAD

**Issue by RobAndrewHurst** [GEOLYTIX/xyz#219]
When Selecting Multiple cluster points, the infoj does not stack and show the multiple infoj's

![image](https://user-images.githubusercontent.com/43169158/69858455-b5a56380-129a-11ea-91c8-2605297d8033.png)

**Comment by dbauszus-glx** [comment]
Could this bug be reproduced?

**Comment by lisataylor-geolytix** [comment]
Yes, it’s the same on the pubs and sites layer within M&B when run locally on development on Friday.... I haven’t tried other configs though

**Comment by dbauszus-glx** [comment]
Rob, can you provide some detail into the nature of the bug?

**Comment by lisataylor-geolytix** [comment]
I can do the non technical version!  When you click on one point everything behaves normally.... clicking on a second or third and the catchments may show but no details are shown in the infoj and no letter highlights the selected location on the map.  This worked in v2.0.0

**Comment by RobAndrewHurst** [comment]
We can replicate this on mb_v2. 

![image](https://user-images.githubusercontent.com/43169158/69941686-ae759400-14ec-11ea-81e2-6458013557ee.png)

Basically there is a null object on the lastchild of the nav_bar in the dataview.

**Comment by cityremade** [comment]
fix in, please test.

**Comment by cityremade** [comment]
@RobAndrewHurst can this be closed now?
